src/pages/ComponentsView/index.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <main>
+    <h1 class="title">Components</h1>
```

Comment:
FYI: Code can be more readable after writing what's in side the tag("Components") from the new line. It's considered a good habit, but it's the preference
I found it here:
https://www.toptal.com/front-end/frontend-clean-code-guide

Reply:
yes, I believe it is a good recommendation, but for the code formatting I use prettier `npm run format`
I tried to do by your idea, but my prettier changed to previous code style
I will try to fix it during our next lessons when we will research how to setup eslint and prettier

thank you for the comment and link with explanations